impeller/entity/contents/filters/filter_contents.h

```diff
@@ -132,17 +132,24 @@ class FilterContents : public Contents {
   // |Contents|
   const FilterContents* AsFilter() const override;
 
-  /// @brief Determines the coverage of source pixels that will be needed
-  ///        to apply this filter under the given transform and produce
-  ///        results anywhere within the indicated coverage limit.
+  /// @brief  Determines the coverage of source pixels that will be needed
+  ///         to produce results for the indicated |output_limit| under the
```

Comment:
What coordinate system is `output_limit` in?

Reply:
I tried to detail the coordinate spaces in the latest version of the comment which has lots of changes, so I'm going to call this one resolved, though my new description may still have some confusion in it...